site/layouts/shortcodes/callout-deprecated-dark-variants.html

```diff
@@ -4,6 +4,6 @@
 
 <div class="bd-callout bd-callout-warning">
   <p>
-    <strong>Heads up!</strong> Dark variants for components were deprecated in v5.3.3 with the introduction of color modes.<br>Add <code>data-bs-theme="dark"</code> to the <code>.{{ $component }}</code> or any ancestor element to enable a component-specific color mode. <a href="/docs/customize/color-modes">Learn more about our color modes</a>.
+    <strong>Heads up!</strong> Dark variants for components were deprecated in v5.3.3 with the introduction of color modes.<br><br>Add <code>data-bs-theme="dark"</code> to the <code>.{{ $component }}</code> or any ancestor element to enable a component-specific color mode. <a href="/docs/customize/color-modes">Learn more about our color modes</a>.
```

Comment:
IMO the previous wording is better since it adds more precise information about the role of the contextual dark mode in this case:

```suggestion
    <strong>Heads up!</strong> Dark variants for components are deprecated in Boosted v5.3.3. They are replaced by our contextual dark mode.<br><br>Add <code>data-bs-theme="dark"</code> to the <code>.{{ $component }}</code> or any ancestor element to enable a component-specific color mode. <a href="/docs/customize/color-modes">Learn more about our color modes</a>.
```